File: tyrant/go/read/tyrant_read.go
// ----------------------------------------------------------------
//
//	tyrant_read.go
//
//					Jan/22/2014
//
// ----------------------------------------------------------------
package main
 
import (
	"fmt"
	"net"
//	"strings"
//	"encoding/json"
)

// ----------------------------------------------------------------
func main () {

	fmt.Println ("*** 開始 ***")

	conn, err := net.Dial ("tcp", "localhost:1978")
	if err != nil {
		fmt.Println(err)
		return
		}

	keys := []string {"t4761","t4762","t4763","t4764","t4765",
			"t4766","t4767","t4768","t4769"}

	for _,value := range keys {
		json_str := mcached_socket_read_proc (conn,value)

		if 0 < len (json_str) {
			json_parser (value,json_str)
			}
	}

	conn.Close ()

	fmt.Println ("*** 終了 ***")
}

// ----------------------------------------------------------------

File: tyrant/ocaml/read/tyrant_read.ml
(* ------------------------------------------------------------------- *)
(*
	tyrant_read.ml

					Feb/08/2012
*)
(* ------------------------------------------------------------------- *)
open String
open Memcache
open Json_type
open Json_type.Browse
open Mcached_manipulate
(* ------------------------------------------------------------------- *)
let _ = print_endline "*** 開始 ***"

let _ = Printf.printf "version: %s\n" Memcache.version

let cache = Memcache.open_connection "localhost" 1978

let keys = ["t4761"; "t4762"; "t4763";"t4764";"t4765";
	 "t4766"; "t4767";"t4768";"t4769"]

let _= out_proc cache keys

let _ =  Memcache.close_connection cache

let _ = print_endline "*** 終了 ***"
(* ------------------------------------------------------------------- *)
